lib/wrappers/rust/Readme.md

```diff
@@ -0,0 +1,26 @@
+# Rust wrapper 
+
+The rust wrapper library wrapping the cw-core library, which is used to control the contour wall. The examples of using rust wrapper is put at the end of the lib.rs file.
+
+# How to run
+
+1. Install Cargo Rust version >=  1.75
```

Comment:
Good addition, however, it can be even better; you can force this. If you add this line `rust-version = "1.75` to `Cargo.toml`, Cargo will force people who want to run this code to have at least version 1.75.